app/javascript/locale/ja.ts

```diff
@@ -0,0 +1,27 @@
+export default {
+  homepage: {
+    referral: {
+      confirmed: "確認済み",
+      details: 'See details for the areas in each group',
+      downloaded: "ダウンロード済み",
+      group: 'グループ',
+      installed: "インストール済み",
+      statement: "この期間の支払いについては、明細を確認してください。",
+      title: "紹介プロモーションの統計"
+    }
+  },
+  settings: {
+    contact: {
+      edit: "編集",
+      email: '新しいメール',
+      error: "Unable to change email; the email address may be in use. Please enter a different email address.",
```

Comment:
We're missing some translations here, but i'm not sure what the proper process would be for this